src/renderer/hooks/useVolume24Price.ts

```diff
@@ -0,0 +1,60 @@
+import * as RD from '@devexperts/remote-data-ts'
+import { getValueOfRuneInAsset } from '@thorchain/asgardex-util'
+import { baseAmount, bnOrZero } from '@xchainjs/xchain-util'
+import * as FP from 'fp-ts/lib/function'
+import { useObservableState } from 'observable-hooks'
+import * as Rx from 'rxjs'
+import * as RxOp from 'rxjs/operators'
+
+import { useMidgardContext } from '../contexts/MidgardContext'
+import { sequenceTRD } from '../helpers/fpHelpers'
+import { PriceRD } from '../services/midgard/types'
+import { AssetWithAmount } from '../types/asgardex'
+import { GetLiquidityHistoryIntervalEnum, GetSwapHistoryIntervalEnum } from '../types/generated/midgard'
+
+export const useVolume24Price = () => {
+  const {
+    service: {
+      pools: { poolsState$, selectedPricePool$, apiGetSwapHistory$, apiGetLiquidityHistory$ }
+    }
+  } = useMidgardContext()
+
+  const [volume24PriceRD] = useObservableState<PriceRD>(
+    () =>
+      FP.pipe(
+        Rx.combineLatest([
+          apiGetSwapHistory$({ interval: GetSwapHistoryIntervalEnum.Day, count: 1 }),
+          apiGetLiquidityHistory$({ interval: GetLiquidityHistoryIntervalEnum.Day, count: 1 }),
+          poolsState$,
```

Comment:
looks like `poolsState$` is unused in this stream at all...

Reply:
It's needed to load pools - in other case default price pool will be "RUNE" based. It will be used for reloading `volume24hrs` as well I think (which will be next PR)